Makefile

```diff
@@ -20,6 +22,12 @@ manager:
 push-manager:
 	docker push $(MANAGER_IMAGE_FULL_NAME)
 
+gen-k8s:
+	operator-sdk generate k8s
+
+unit-test:
+	ginkgo $(GINKGO_ARGS) ./pkg/apis/nmstate/v1
```

Comment:
as discussed, let's try to replace this with `go test` and drop `go get ` on ginkgo and gomega

Reply:
Looks like ginkgo has nice features to randomize tests, let's keep it go get is just taking the command line tool but I think we are compiling towards vendoring one.